src/Tile.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+function Tile ({title, paragraph, image, image_name}) {
+    return (
+        <section>
+            <h2>{title}</h2>
+            <p>{paragraph}</p>
+            <img src={image} alt={image_name}/>
+        </section>
+    )
+}
+
+export default Tile;
```

Comment:
Hier had je ook children i.p.v. <p> kunnen gebruiken maar dat is wat je persoonlijk fijn vindt.